_data/schedule.yaml

```diff
@@ -153,46 +191,61 @@
   summary: "Symbolic links, hard links, directory searching, intro to permissions"
   resources: "See <a href='https://github.com/angrave/SystemProgramming/wiki/File-System%2C-Part-3%3A-Permissions'>Permissions</a>."
   color: "#FFC107"
+  number: 31
 -
   title: "Files-4"
   summary: "File permissions, directories, file globbing, intro to RAID"
   resources: "See <a href='https://github.com/angrave/SystemProgramming/wiki/File-System%2C-Part-3%3A-Permissions'>Permissions</a>."
   color: "#FFC107"
+<<<<<<< HEAD

```

Comment:
Is it fine to leave these merge messages in here?

Reply:
Not okay: I fixed this post merge.

Reply:
Not okay: I fixed this post merge.